content/applications/hr/payroll/contracts.rst

```diff
@@ -2,224 +2,361 @@
 
 Any automatic deductions or allocations for an employee, such as child support payments and wage
-garnishments, are referred to as a *salary attachment*. This section is where all of these
-deductions or allocations are set.
+garnishments, are referred to as a *salary attachment*. Navigate to :menuselection:`Payroll app -->
+Contracts -> Salary Attachments` to view a list of all the currently configured salary attachments.
+
+Each salary attachment appears with all relevant details displayed. The :guilabel:`Status` for each
+attachment is color-coded in the far right column. Currently running salary attachments have a
+:guilabel:`Status` of :guilabel:`Running`, and appear in green. Salary attachments that have been
+paid in full and are no longer active have a :guilabel:`Status` of :guilabel:`Completed`, and appear
+in blue. Cancelled salary attachments have a :guilabel:`Status` of :guilabel:`Cancelled` and appear
+in red.
 
-To add a new deduction, click :guilabel:`Add a line`. Type in a description for the allocation under
-:guilabel:`Description`.
+.. image:: contracts/attachments.png
+   :align: center
+   :alt: A list view of all the salary attachments with their status displayed.
+
+New salary attachment
+---------------------
+
+To create a new salary attachment, click :guilabel:`New`, and a blank salary attachment form loads.
+Enter the following information on the form:
+
+- :guilabel:`Employees`: add any employees the salary attachment applies to using the drop-down
+  menu.
+
+  .. tip::
+     Multiple employees can be added if the salary attachment details are identical. After all
+     employees are added, a :guilabel:`Create Individual Attachments` button appears at the top of
+     the form.
+
+     After the form is completed, click the :guilabel:`Create Individual Attachments` button to
+     create separate salary attachments for each of the employees listed in this field.
+
+     .. image:: contracts/individual-attachments.png
+        :align: center
+        :alt: The Create Individual Attachments button that appears after multiple employees are
+              added to the Employees field.
+
+     This is a time-saving tip so that separate salary attachments do not need to be created
+     individually, but can be created in a batch using this method.
+
+- :guilabel:`Description`: enter a description for the specific type of salary attachment.
+- :guilabel:`Type`: select the type of salary attachment from the drop-down menu. The options listed
+  come from the salary attachment types configured in the configuration menu.
+- :guilabel:`Start Date`: enter the date the salary attachment begins. The first of the current
+  month populates this field by default. Click on the date and a calendar appears. Navigate to the
+  desired month and year by using the :guilabel:`< > (arrow)` icons, and click on the date to select
+  it.
+- :guilabel:`Estimated End Date`: this field only appears after the :guilabel:`Monthly Amount` field
```

Comment:
the placement of this breakdown was a bit confusing. I fully understand that this is where it appears on the form -- but it only appears if a later field is modified. That said, I think it should be mentioned after the Monthly Amount field, so as to avoid any confusion with the reader

Reply:
OK thanks - I was going to at first, but then thought it should be where it was in order on the form- so I am glad you also think it should go after the item that makes it appear. Moved!